gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,8 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.3-all.zip
```

Comment:
locally not running anymore. Maybe on CI or someone can fix the missing part, thank you.


```
* Where:
Build file '/Users/vincent.potucek/IdeaProjects/sonarqube/build.gradle' line: 712

* What went wrong:
A problem occurred evaluating root project 'sonarqube'.
> Failed to apply plugin 'com.google.protobuf'.
   > Could not get unknown property 'convention' for project ':sonar-core' of type org.gradle.api.Project.
```

Reply:
Moving from Gradle 8 to Gradle 9 has to be checked properly with our full build process.
Very often plugins break (like protobuf in this case) and the whole operation has to be planned.